src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -140,28 +142,39 @@
     /**
      * By default, even filtered records are validated.
      */
-    @Argument(fullName = "doNotValidateFilteredRecords", shortName = "doNotValidateFilteredRecords", doc = "skip validation on filtered records", optional = true)
+    @Argument(fullName = "doNotValidateFilteredRecords", shortName = "doNotValidateFilteredRecords", doc = "skip validation on filtered records", optional = true, mutex = "validateGVCF")
     Boolean DO_NOT_VALIDATE_FILTERED = false;
 
     @Argument(fullName = "warnOnErrors", shortName = "warnOnErrors", doc = "just emit warnings on errors instead of terminating the run at the first instance", optional = true)
     Boolean WARN_ON_ERROR = false;
 
     /**
+     *  Validate this file as a gvcf. In particular, every variant must include a <NON_REF> allele, and that
+     *  every base in the territory under consideration is covered by a variant (or a reference block).
+     *  If you specifed intervals (using -L or -XL) to restrict analysis to a subset of genomic regions,
+     *  those intervals will need to be covered in a valid gvcf.
+     */
+    @Argument(fullName = "validateGVCF", shortName = "gvcf", doc = "Validate this file as a GVCF", optional = true, mutex = "doNotValidateFilteredRecords")
```

Comment:
Could you pull this argument name out as a public constant in the class and then use the constant since it's referenced in two places now?  Likewise with the doNotValidateFilteredRecords argument name.